com.civicrm.hrjobroles/views/include/job_role_panel.html

```diff
@@ -64,7 +64,7 @@
                 </div>
                 <div class="form-group" ng-init="jobroles.initData(job_roles_data.id, 'job_contract_id', job_roles_data.job_contract_id)">
                   <label for="contract" class="col-sm-4 control-label">Contract:</label>
-                  <div class="col-sm-8">
+                  <div class="col-sm-6">
```

Comment:
yeah it's a bit complicated, because of the grid system. The columns width had changed but visually (the end result) is kept the same between the old and new theme